### PR TITLE
Introduction page

### PR DESCRIPTION
**Pull request by k-rahul21**

### CODE REVIEW

File: css/sidebar.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+.sidebar {
+  position: fixed;
+  width: 200px;
+  height: 500px;
```
Comment (sanket4120) [inline css/sidebar.css]: Can give sidebar height relative to viewport height so that it will look good on both larger and smaller screen devices.

**Review by sanket4120 — APPROVED** [review]
Rest looks good to me

**Review by Snigdha17 — APPROVED** [review]
Change looks good to be merged

**Review by rahulcs754 — COMMENTED** [review]
code is good and clean

**Pull request merged**